src/components/Books.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import Form from './Form';
+
+export default function Books() {
+  const [bookList] = useState({ title: 'Transformation', author: 'Franz Kafka     ' });
+  const { title, author } = bookList;
+
+  return (
+    <>
+
+      <div className="bookList">
+        <h2> List of Books</h2>
+        <ul>
+          <li className="bookItems">
+            <p>{title}</p>
+            <p>{author}</p>
+            <button type="button" className="button key">Remove</button>
+          </li>
```

Comment:
- [ ] Great job structuring your components. :+1: Let's work on some more details to achieve the project requirements. It is suggested that we should utilize a component that takes props to render an individual Book. Kindly extract these lines to a new component and pass the data through props. ✍️

**Project Requirements**
![image](https://user-images.githubusercontent.com/17243325/222685454-3153fe44-10fa-4641-8121-a6b1a9e6a8b8.png)

Reply:
Thank you, it is very useful way :)